osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/metrics/DefaultMetricsService.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.shared.metrics;
+
+import org.opensmartgridplatform.shared.config.PrometheusDisabledCondition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.context.annotation.Conditional;
+import org.springframework.stereotype.Component;
+
+@Component
+@Conditional(PrometheusDisabledCondition.class)
+public class DefaultMetricsService implements MetricsService {
```

Comment:
Perhaps some other name would better describe what this does. Default may suggest some kind of metrics related working, but this ignores the metering part.
Another thing that might be a little confusing is that this `DefaultMetricsService` is not related to `enableDefaultMetrics` in `MetricsConfig`.

Reply:
Decided to not follow this path, class is no longer part of the solution